crates/host/src/wasmbus/mod.rs

```diff
@@ -1681,14 +1797,16 @@ impl Host {
             lattice_prefix: self.host_config.lattice_prefix.clone(),
             origin,
             cluster_key: Arc::clone(&self.cluster_key),
+            claims: claims.clone(),
             links: Arc::new(RwLock::new(links)),
             targets: Arc::new(RwLock::default()),
+            host_key: Arc::clone(&self.host_key),
         };
 
         let pool = ActorInstancePool::new(actor.clone(), Some(count));
         let instances = self
             .instantiate_actor(
-                claims,
+                claims.clone(),
```

Comment:
Do you need to clone here? (Same question on all the places we call `instantiate_actor`)

Reply:
I was able to change it to a reference because of the cloning we do inside the block